data/data/bootstrap/files/usr/local/bin/wait-for-ha-api.sh

```diff
@@ -13,6 +27,32 @@ wait_for_ha_api() {
     done
 }
 
+
+get_topology() {
+    output=$(oc --kubeconfig="$KUBECONFIG" get infrastructures cluster -o jsonpath='{.status.controlPlaneTopology}' 2>&1 )
+    # shellcheck disable=SC2124
+    status=$?
+    if [[ $status -ne 0 ]]
+    then
+	echo "The following error happened while retrieving infrastructures/cluster object"
+	echo "$output"
+	return 1
+    fi
+
+    if [ -z "$output" ]
+    then
+	echo "status.infrastructureTopology of the infrastructures/cluster object is empty"
+	return 1
+    fi
+
+    if [ "$output" == "SingleReplica" ]
```

Comment:
This mix of Bourne syntax (`[`) and Bash syntax (`==`) makes my eye twitch.  * grin *.  Might as well use `[[` here to be consistent with the use of `[[` elsewhere in the file and the use of Bash-specific `==`.